.github/workflows/pythonpackage.yml

```diff
@@ -23,7 +23,7 @@ jobs:
           - "pypy-3.8"
           - "pypy-3.9"
         include:
-        - python-version: "3.11.0-alpha - 3.11"
```

Comment:
Hum, i did not grasp that the `-` was to specify a range.
But it seems to me that `3.11-dev` should do the same as the README says
> Use -dev instead of a patch number (e.g., 3.11-dev) to install the latest release of a minor version, alpha and beta releases included.

Reply:
Hum, i did not grasp that the `-` was to specify a range.
But it seems to me that `3.11-dev` should do the same as the README says
> Use -dev instead of a patch number (e.g., 3.11-dev) to install the latest release of a minor version, alpha and beta releases included.